### ISSUE TITLE
Feature request: attach to existing run?

### ISSUE THREAD

**Issue by shosti** [rainforestapp/rainforest-cli#98]
Right now (afaik) there are 2 options for running tests: in the background (the default) or in the foreground (`--fg`). It would be nice if it were possible to "attach" to an existing run (i.e. like `--fg` but for a run that's already in progress--it would exit immediately with the corresponding exit code for runs that were already done).  That way, we could set off a run at the beginning of our CI process and "attach" after the unit tests finish, so the unit tests and the Rainforest tests run in parallel.

@epaulet thoughts?
